package.json

```
@@ -77,7 +77,11 @@
     "transform:android": "node ./examples/android/build.js",
     "transform:ios": "node ./examples/ios/build.js",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "preinstall": "npx npm-force-resolutions"
```

I wonder if this is a potential security issue? It'll run whichever the most recent version is so if they release a malicious version it will get run on CI.

Instead, we could add it as a `devDependency` and change the script to
```suggestion
    "preinstall": "npm-force-resolutions"
```

---

Also, this has completely changed the `package-lock.json` file. Do we know if it's going to be stable across runs?

---

I've just replaced this with [`overrides` feature of `npm`](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides).